srcpkgs/gnirehtet/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'gnirehtet'
+pkgname=gnirehtet
+version=2.4
+revision=1
+build_wrksrc=relay-rust
+build_style=cargo
+makedepends="android-tools"
```

Comment:
does it need `adb` at run-time? if true, this is missing `depends="android-tools"`